modules/spo-binding-workloads.adoc

```diff
@@ -0,0 +1,96 @@
+// Module included in the following assemblies:
+//
+// * security/security_profiles_operator/spo-seccomp.adoc
+// * security/security_profiles_operator/spo-selinux.adoc
+
+ifeval::["{context}" == "spo-seccomp"]
+:seccomp:
+:type: seccomp
+:kind: SeccompProfile
+endif::[]
+ifeval::["{context}" == "spo-selinux"]
+:selinux:
+:type: SELinux
+:kind: SelinuxProfile
+endif::[]
+
+:_content-type: PROCEDURE
+[id="spo-binding-workloads_{context}"]
+= Binding workloads to profiles with ProfileBindings
+
+You can use the `ProfileBinding` resource to bind a security profile to the `SecurityContext` of a container.
+
+.Procedure
+
+. To bind a `Pod` that uses an `nginx:1.19.1` image to the example `{kind}` profile, create a `ProfileBinding` object in the same namespace with the `Pod` and the `{kind}` objects:
++
+[source,yaml,subs="attributes+"]
+----
+apiVersion: security-profiles-operator.x-k8s.io/v1alpha1
+kind: ProfileBinding
+metadata:
+  namespace: my-namespace
+  name: nginx-binding
+spec:
+  profileRef:
+    kind: {kind}
+    name: profile
+  image: nginx:1.19.1
```

Comment:
Please update nginx:1.19.1 to quay.io/security-profiles-operator/test-nginx-unprivileged:1.21. Same reason as https://github.com/openshift/openshift-docs/pull/52698/files#r1044168420.